utils/a11y_controller.js

```diff
@@ -22,7 +22,10 @@ export default class A11yController {
 
         this.mouseIsPressed = false;
 
-        this.lastInputEventIsKeyboard = false;
+        this.lastInputEventIsKeyDown = false;
+        this.lastInputEventIsKeyboard = true;
```

Comment:
The difference between these two is that the a11y-controller's default behaviour is to only show a visible focus if the user is literally in the middle of using the keyboard (i.e. the user is pressing a key on the keyboard). But we want to be able to manually focus elements even if no keys are currently pressed. So, we need a distinction between these two actions.

Reply:
Might be good to add this as a comment